### PR TITLE
GEODE-7109: add test case that multiple sessions can be maintained.

### PR DESCRIPTION
**Pull request by pivotal-eshu**
* work around an known issue by verifying after HARegionQueue is drained.

### CODE REVIEW

File: geode-assembly/src/distributedTest/java/org/apache/geode/session/tests/TomcatTest.java [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+  }
+
+  private void verifySessionGets(Client client, HashMap<String, String> attributes,
+      Client.Response resp, int operatingContainer) throws IOException, URISyntaxException {
+    int count = 0;
+    int whichOne = random.nextInt(attributes.size());
+    Iterator iterator = attributes.entrySet().iterator();
+    while (iterator.hasNext()) {
+      Map.Entry entry = (Map.Entry) iterator.next();
+      if (count == whichOne) {
```
Comment (jchen21) [inline geode-assembly/src/distributedTest/java/org/apache/geode/session/tests/TomcatTest.java]: Why not verify all the entries in `attributes`?

File: extensions/session-testing-war/src/main/java/org/apache/geode/modules/session/CommandServlet.java [hunk 2]
```diff
@@ -92,10 +109,47 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
...
+    Execution execution = FunctionService.onServers(cache);
+
+    ResultCollector collector = execution.execute(GetQueueSize.ID);
+    List list = (List) collector.getResult();
+    for (Object object : list) {
+      for (Object queue : ((Map) object).keySet()) {
+        manager.getLogger().info("client cache has queue: " + queue);
+      }
+    }
+    for (Object object : list) {
```
[the remark this answers is not in the document]
Comment (pivotal-eshu) [reply extensions/session-testing-war/src/main/java/org/apache/geode/modules/session/CommandServlet.java]: You are right for this. However, the original intention was to only log the queue names once. Code has been changed to achieve this.

File: geode-assembly/src/distributedTest/java/org/apache/geode/session/tests/TomcatTest.java (hunk 1 above)
Comment (pivotal-eshu) [reply geode-assembly/src/distributedTest/java/org/apache/geode/session/tests/TomcatTest.java]: All attributes are verified at the end of test. No need to verify the same attributes after each operation (basically verify same attributes again.) Test will just randomly check one of the attributes and verify all at the end of the test.

**Comment by onichols-pivotal** [comment]
this PR appears to be abandoned, can it be closed?

**Comment by onichols-pivotal** [comment]
closed due to lack of activity

**Pull request closed without merging**